docs/docs/genai/eval-monitor/scorers/llm-judge/migration-guide.mdx

```diff
@@ -0,0 +1,570 @@
+import { APILink } from "@site/src/components/APILink";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import TileCard from "@site/src/components/TileCard";
+import TilesGrid from "@site/src/components/TilesGrid";
+import { MessageSquare, BookOpen, Rocket } from "lucide-react";
+
+# Migrating from Legacy Judges to make_judge
+
+:::warning[Deprecation Notice]
+
+The legacy judge functions (`is_context_relevant`, `is_correct`, `is_grounded`, `is_context_sufficient`, `meets_guidelines`, and `custom_prompt_judge`) are deprecated as of MLflow 3.4.0 and will be removed in a future release. This guide helps you migrate to the new, more flexible `make_judge` API.
```

Comment:
There is a different way to describe the changes:

1. We are introducing `make_judge()` as the generic way to create Scorers that can be directly plugged into evaluation and monitoring. This is meant to deprecate `custom_prompt_judge()`. 
2. We are re-defining `Correctness`, `Groundedness`, ... as calls to `make_judge()`. 
3. We are retiring the `is_correct`, ... functions in favor of their corresponding scorers.

I think the phrasing of 3 is important -- we are not asking customers to migrate `is_context_relevant()` to a call to `make_judge()`, but to use the `ContextRelevance()` scorer. This makes the migration story easier as well. 

Thoughts?